ws2812b/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ Copyright 2016 Google Inc.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.google.android.things.contrib.driver.ws2812b">
+    <application>
+        <uses-library android:name="com.google.android.things"/>
```

Comment:
drivers in this repo now have required=false

`<uses-library android:required="false" android:name="com.google.android.things"/>`

to allow for any application using them to not be filtered from the playstore (if for example they wrote an app that targets phones and an IoT device)

Reply:
Fixed: https://github.com/androidthings/contrib-drivers/pull/76/commits/bb7b9361d825282451b1861f1924080ac7a74abc